### PR TITLE
Document Linux capture methods and nvfbc patching

### PR DESCRIPTION
**Pull request by KuleRucket**
## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I want maintainers to keep my branch updated

### CODE REVIEW

File: docs/source/about/installation.rst [hunk 1]
```diff
@@ -21,13 +21,36 @@ See :ref:`Docker <about/docker:docker>` for additional information.
 
 Linux
 -----
-Follow the instructions for your preferred package type below.
+Capture Methods
```
Comment (ReenigneArcher) [inline docs/source/about/installation.rst:24]: I'm not sure I understand why this was added to the installation section... or why some of the CUDA comments were removed.

**Review by ReenigneArcher — CHANGES_REQUESTED** [review]

File: docs/source/about/installation.rst (hunk 1 above)
Comment (KuleRucket) [reply docs/source/about/installation.rst:24]: The sections that follow mention NvFBC, KMS etc without actually explaining what they mean. I could not find a better place for it.

No information has been removed, only extended. "CUDA is used for NVFBC capture." => "This method requires CUDA as the frame is captured to cuda memory."

Comment (ReenigneArcher) [reply docs/source/about/installation.rst:24]: The title for the cuda compatibility table was removed... Additionally, this (`.. Tip:: See `CUDA GPUS <https://developer.nvidia.com/cuda-gpus>`_ to cross reference Compute Capability to your GPU.`) had the [admonition](https://pradyunsg.me/furo/reference/admonitions/) removed.

That table should not be within the `Packages` section. Perhaps should be given it's own section.

I think the capture methods should go under `usage` though as it has nothing to do with installation.

You can see the compiled docs for your PR here: https://lizardbyte--888.org.readthedocs.build/projects/sunshine/en/888/about/installation.html#linux

Comment (KuleRucket) [reply docs/source/about/installation.rst:24]: It's still there. As part of the section.

**Pull request closed without merging**